tests/integration/taskManagement/taskManagement.spec.ts

```diff
@@ -254,7 +254,7 @@ describe('tasks', function () {
 
         const response = await requestSender.abortJobAndTasks(jobId);
 
-        expect(response.status).toBe(httpStatusCodes.NO_CONTENT);
+        expect(response.status).toBe(httpStatusCodes.OK);
         expect(jobRepositoryMocks.saveMock).toHaveBeenCalledTimes(1);
```

Comment:
you added specific message for every action test that the correct value is returned

Reply:
fixed